client/state/plugins/installed/actions.js

```diff
@@ -51,6 +51,25 @@ import { getSite } from 'calypso/state/sites/selectors';
 
 import 'calypso/state/plugins/init';
 
+/**
+ * Determines the truthiness of a site specific property regardless of whether it is on the plugin object
+ * or on one of the plugin's site objects.
+ *
+ * @param {string} prop - The site property to check. One of 'active', 'autoupdate', 'update', or 'version'.
+ * @param {Object} plugin - The plugin object
+ * @param {number} siteId - The ID of the site if it exists
```

Comment:
If it exists? This adds some confusion, since the ID is expected to always be a number. Is the idea that we might have the site ID but the plugin data for the site may be missing?

Reply:
Just bad wording on my part. I removed it.